pep-0001.txt

```diff
@@ -282,22 +282,22 @@ notifying the PEP author(s) and giving them a chance to make revisions.
 The final authority for PEP approval is the Steering Council. However, whenever
 a new PEP is put forward, any core developer that believes they are suitably
 experienced to make the final decision on that PEP may offer to serve as
-the PEP-Delegate for that PEP, and they will then have the
-authority to approve (or reject) that PEP. 
-
-The term "PEP-Delegate" is used under the Steering
-Council governance model. The PEP's designated decision maker,
-the "PEP-Delegate" with the Steering Council's support, is
-recorded in the "PEP-Delegate" field in the PEP's header. For
-PEPs written prior to the Steering Council's governance model, the field name
+the PEP-Delegate for that PEP, and not declined by the Steering Council,
```

Comment:
```suggestion
the PEP-Delegate for that PEP, and they should notify the Steering Council of their offer. If not declined by the Steering Council,
```

Reply:
Since the notification requirements are discussed more specifically two paragraphs below this, I'm not sure we need to discuss that in detail here, where we're just summarizing the basic nature of the PEP-Delegate position rather than how to become one (as discussed below). However, I rewrote it to incorporate the spirit of this, specifying that the steering council "assents to their offer" (which requires that the Steering Council be notified of said offer), and I also broke this up into two sentences as you had it, and fixed the missing word ("if"), as well as some repetition and suboptimal phrasing I noticed in the process. Is that okay?

Reply:
In general, I think that is fine. I would be explicit about notification of the Steering Council is one step toward becoming the PEP-Delegate. Acceptance by the steering council is the second step.

Reply:
Looks good to me.